Trend Micro/trend-micro-vision-one-oat/ingest/parser.yml

```diff
@@ -82,3 +83,21 @@ stages:
               {%- if "." in item -%}{%- set ids = ids.append(item) -%}{%- endif -%}
             {%- endfor -%}
             {%- if ids | length > 0 -%}{{ ids | tojson }}{%- endif -%}
+
+      - set:
+          threat.tactic.id: "{{parsed_event.message.filters | map(attribute='mitreTacticIds') | list | sum(start = [])}}"
```

Comment:
I think we can move `threat.technique.subtechnique.id` and `threat.technique.id` in this action as they also `parsed_event.message.filters`.